src/Func.h

```diff
@@ -210,12 +210,17 @@ class ScriptFunc final : public Func {
 	 */
 	bool StrengthenClosureReference(Frame* f);
 
+	/**
+	 * Whether the function's closure uses copy semantics.
+	 */
+	virtual bool CopySemantics() const;
```

Comment:
I think a better name is `HasCopySemantics()` since it's more clear that it's checking for something.

Reply:
Agreed.